### ISSUE TITLE
MonthInput.render(): A valid React element (or null) must be returned. You may have returned undefined, an array or some other invalid object.

### ISSUE THREAD

**Issue by ThomasODev** [wojtekmaj/react-datetime-picker#16]
Hey,

I'd like to use datetime-picker in my project but I always get this error.
`MonthInput.render(): A valid React element (or null) must be returned. You may have returned undefined, an array or some other invalid object.`

I'm working on react 15.4.2 so as you mentioned I used v.1.0.1.
I've seen similar issues and someone said the calendar should be installed. I did that (v.2.7.0) but also got the error.

Because you already encountered this issue, do you have updates about it?

Thanks,

-Thomas

**Comment by wojtekmaj** [comment]
Hey there,
I suggest you updating `react` to 15.5 at the very least. Looking at the lockfile, I have stated 15.5 as the minimum requirement version. Docs were wrong on this - I updated them. Thanks!

If that won't help, check your node_modules (or lockfile) how many copies of these you have installed now:

* react
* react-calendar

If you have more than two of any of these, we should investigate why that happened
